### DISCUSSION TITLE
Broke Netbox when we updated the firewall

### DISCUSSION THREAD

**Discussion by getschools** [netbox-community/netbox#17580] [Help Wanted!]
When we updated our firewall and the default gateway changed, we lost access to Netbox.  How do I regain access?

**Comment by MattDRogers** [comment]
If you can access the VM or server your instance of NetBox is running on, you should be able to console/KVM in and update your system's IP configuration I would think.
Without more to go on, that's the best advice I have.

**Reply by getschools** [reply]
I did sign into the host linux server and update the IP configuration.  This did not correct the issue.  Not sure where that info would need to be changed in addition where I have already changed it.

**Reply by a084ed22** [reply]
No change on netbox's side should be required in the even of a change of the default gateway for the host it is running on. If you are having problems, this is likely an unrelated issue with your infrastructure, and not with netbox itself.
